From 6e526cacbd163f10db5cda62e10744f31d2cb5ad Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Tue, 7 Mar 2023 10:47:38 -0500
Subject: [PATCH] Patch out maximum Python version check

Fedora must build this package with the current version of Python,
whether upstream is ready or not.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 02b1604..b08653d 100644
--- a/setup.py
+++ b/setup.py
@@ -46,7 +46,7 @@ def _guard_py_ver():
     max_py = parse(max_python_version)
     cur_py = parse('.'.join(map(str, sys.version_info[:3])))
 
-    if not min_py <= cur_py < max_py:
+    if not min_py <= cur_py:
         msg = ('Cannot install on Python version {}; only versions >={},<{} '
                'are supported.')
         raise RuntimeError(msg.format(cur_py, min_py, max_py))
-- 
2.39.2

